tests/typecheck/managers/test_managers.yml

```diff
@@ -350,11 +350,12 @@
         -   path: myapp/__init__.py
         -   path: myapp/models.py
             content: |
+                from typing import Any
                 from django.db import models
                 class MyManager(models.Manager):
                     def get_instance(self) -> int:
                         pass
-                    def get_instance_untyped(self, name):
+                    def get_instance_untyped(self, name: str) -> Any:
```

Comment:
Looks like it contradicts its name now. Maybe `type: ignore[error-code]` it instead?

Reply:
OK, I'll need to add `# type: ignore[no-untyped-def]` in the other PR, because this check isn't enabled here yet.